internal/services/datafactory/azuresdkhacks/pipelines.go

```diff
@@ -0,0 +1,208 @@
+package azuresdkhacks
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/azure"
+	"github.com/Azure/go-autorest/autorest/validation"
+	"github.com/tombuildsstuff/kermit/sdk/datafactory/2018-06-01/datafactory"
+)
+
+// TODO4.0: check if the workaround could be removed.
+// Workaround for https://github.com/hashicorp/terraform-provider-azurerm/issues/24758
+// Tracked on https://github.com/Azure/azure-rest-api-specs/issues/27816
+
```

Comment:
If this file is just a copy paste of the original implementation, we shall also just write down something like: "This file is just a copy paste from file: ...".